_data/mohamedelraghy/mohamedelraghy.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mohamed EL-Raghy # No longer than 28 characters
+institution: Suez Canal university 🚩 # no longer than 58 characters
+profile_pic: Mohamed.jepg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Don't forget to change this as well to `Mohamed.jpg` after renaming your image file

Reply:
@mitulsavani Do you think the flag icon would cause troubles as it's a non ASCII char?

Reply:
@aryanrawlani28 that shouldn't cause any problems. We have merged some PR that has the flags